birdhouse/deployment/deploy.sh

```diff
@@ -50,12 +53,10 @@
 #   are re-read.  docker-compose is not aware of any changes outside of the
 #   docker-compose.yml file.
 
-
-GREEN=$(tput setaf 2)
-YELLOW=$(tput setaf 3)
-RED=$(tput setaf 1)
-NORMAL=$(tput sgr0)
-
```

Comment:
Why remove this?
Makes it easier to spot important items in the console that often spit out too much text to be able to read all.

Reply:
See why console coloring are removed in this commit message https://github.com/bird-house/birdhouse-deploy/pull/39/commits/a2dad6214b8a4786f4f9b25e0176eaad5e181ecf